CHANGES.md

```diff
@@ -1,5 +1,31 @@
 # Changes
 
+## edge-20.7.1
+
+This edge release introduces the option to persist prometheus data to a volume,
+instead of memory, so that historical metrics are available when prometheus is
+restarted. Also in this release are improvements to control plane check
+conditions for CLI commands, a fix to correctly resolve the CNI image version
+when using helm, support for using URLs with add-ons, proxy improvements under
+concurrency, and functionality to configure the proxy log format by annotation
+or in the helm chart.
```

Comment:
This whole thing feels redundant to me. I'd just omit it (if there's nothing particularly notable about the edge).